dune-project

```
@@ -45,7 +49,11 @@ possible and does not make any assumptions about IO.
   menhir
   ocaml-syntax-shims
   ppx_yojson_conv_lib
-  (ocamlformat :with-test)
-  (ocamlfind (>= 1.5.2))
-  (ocaml (>= 4.06))
-  (dune (>= 2.0))))
+  (ocamlformat
+   (and :dev :with-test))
```

```suggestion
   (or :dev :with-test))
```

We need it both for tests and for dev

---

Done with https://github.com/ocaml/ocaml-lsp/pull/171/commits/7554ae21cb3879ff8a37dc40189913a4fe12ac20